openshift-ci/README.md

```diff
@@ -0,0 +1,31 @@
+# Model Registry Deployment and Deployment Test
+
+This deployment of model-registry is deployed via Opendatahub and used the ODH nightly images deployed to an openshfit cluster.
```

Comment:
consider putting an hyperlink to the OpenShift CI "evironment"?
Somewhere where we could see previous runs or the likes?

This way people not familiar yet with OpenShift CI could better understand which CI system is being discussed here